Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11
+
+WORKDIR /usr/src/app
+
+ADD requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
This is failing with both `docker build .` and `docker buildx build --platform linux/amd64 .` on mac arm64.

```
#0 187.1   Building wheel for pyodbc (setup.py): finished with status 'error'
#0 187.2   error: subprocess-exited-with-error
#0 187.2
#0 187.2   × python setup.py bdist_wheel did not run successfully.
#0 187.2   │ exit code: 1
#0 187.2   ╰─> [14 lines of output]
#0 187.2       running bdist_wheel
#0 187.2       running build
#0 187.2       running build_ext
#0 187.2       building 'pyodbc' extension
#0 187.2       creating build
#0 187.2       creating build/temp.linux-x86_64-cpython-311
#0 187.2       creating build/temp.linux-x86_64-cpython-311/src
#0 187.2       gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DPYODBC_VERSION=4.0.34 -I/usr/local/include/python3.11 -c src/buffer.cpp -o build/temp.linux-x86_64-cpython-311/src/buffer.o -Wno-write-strings
#0 187.2       In file included from src/buffer.cpp:12:
#0 187.2       src/pyodbc.h:56:10: fatal error: sql.h: No such file or directory
#0 187.2          56 | #include <sql.h>
#0 187.2             |          ^~~~~~~
#0 187.2       compilation terminated.
#0 187.2       error: command '/usr/bin/gcc' failed with exit code 1
#0 187.2       [end of output]
```

Reply:
Fixed with extra install